rasa/validator.py

```diff
@@ -64,9 +65,14 @@ def verify_intents(self, ignore_warnings: bool = True) -> bool:
 
         nlu_data_intents = {e.data["intent"] for e in self.intents.intent_examples}
 
-        for intent in self.domain.intents:
+        # Exclude the default intents that are always implicitly present
+        for intent in [
+            item
+            for item in self.domain.intents
+            if item not in constants.DEFAULT_INTENTS
+        ]:
             if intent not in nlu_data_intents:
-                logger.debug(
+                rasa.shared.utils.io.raise_warning(
```

Comment:
I've noticed that there are a couple of others with `debug` messages. In my opinion we should be consistent and have them all as warnings, what do you think?

Reply:
Yes, I've noticed the same thing, but I feel like not all of them are clear cut - which actually  should be more of a warning, and which are really purely for debug purposes? Happy to go through with somebody who's in deeper already than me.

Reply:
Thanks for changing the to warning as well 👍🏻 Makes way more sense now 🎉